cordova-plugin-outline/apple/src/OutlineSentryLogger.swift

```diff
@@ -125,14 +98,14 @@ class OutlineSentryLogger: DDAbstractLogger {
         // Order log lines descending by time.
         let logLines = logContents.components(separatedBy: "\n").reversed()
         for line in logLines {
-          if numBreadcrumbsAdded >= maxNumVpnExtensionBreadcrumbs {
+          if numBreadcrumbsAdded >= OutlinePlugin.kMaxBreadcrumbs / 2 {
```

Comment:
We no longer care about the total VPN + app breadcrumbs?

Reply:
They simplified and removed a lot of things from the API: https://docs.sentry.io/platforms/apple/guides/macos/migration/

As far as I can tell, there's no longer an accessible "client" and I don't have access to the current breadcrumbs anymore, so I can't count them.

Reply:
Got it, thanks